ietf/ietfauth/views.py

```diff
@@ -113,35 +113,48 @@ def index(request):
 #     return HttpResponseRedirect(redirect_to)
 
 def create_account(request):
-    to_email = None
+    new_account_email = None
 
-    if request.method == 'POST':
+    if request.method == "POST":
         form = RegistrationForm(request.POST)
         if form.is_valid():
-            to_email = form.cleaned_data['email'] # This will be lowercase if form.is_valid()
-
-            if User.objects.filter(username=to_email).exists():
-                send_account_creation_exists_email(request, to_email)
+            new_account_email = form.cleaned_data[
+                "email"
+            ]  # This will be lowercase if form.is_valid()
+
+            user = User.objects.filter(username=new_account_email).first()
+            if user:
```

Comment:
This is the test that needs changing, not just choosing the email we send to.
We should enter the `if` code below if a matching User object exists _or_ a matching Email object exists.

Reply:
So something like this?
```python
            existing_user = User.objects.filter(username=new_account_email).first()
            existing_email = Email.objects.filter(email=new_account_email).first()
            if existing_user or existing_email:
                email = (
                    existing_user.person.email_set.filter(active=True).order_by("-primary", "-time").first()
                    if existing_user
                    else new_account_email
                )
                send_account_creation_exists_email(request, new_account_email, email)
```